forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js

```diff
@@ -146,6 +177,34 @@ const Edit = React.memo((props) => {
     return <Loading />;
   }
 
+  const onApplicationFormSubmit = (actionType = "") => {
```

Comment:
I'm wondering if we can change this to be decoupled from the form "submit" button and instead be triggered through a Form Event to match how this happens on the Task page (ServiceTaskDetails.js -> onCustomEventCallBack). That way we can easily disable / enable this per form as needed and also have the option to pass along custom data to the Camunda Task you want to progress. You can check the "Submit Decision" button on the Telework form for that logic

```
  form.emit('customEvent', {
            type: "actionComplete",   
            component: component,
            actionType:data.action
      }); 
```

Reply:
And if you're able to add the ticket number to the title of the PR, that would be great! Just makes it easier to reference this if needed in the future

Reply:
addressed.